kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -179,6 +183,11 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/props', '
         return getJobString(jobState, 'action');
     }
 
+    const validStatusesForAction = {
+        cancel: ['created', 'estimating', 'queued', 'running'],
+        retry: ['created', 'estimating', 'queued', 'running', 'error', 'terminated'],
```

Comment:
allow retries on any extant job that isn't `completed` (in case the front end is out of sync with ee2)